src/get-started/codelab.md

```diff
@@ -539,7 +535,7 @@ at the following link to get back on track.
 
 * [lib/main.dart]({{site.codelab-code-url}}/startup_namer/4_end_of_infinite_list/lib/main.dart)
 
-{% include_relative _app-figure.md image="startup-namer.gif" caption="The app from part 2" class="float-right" %}
+{% include_relative _app-figure.md image="startup-namer.gif" caption="The app from part 2" %}
```

Comment:
We've lost the `class="float-right"`, so the image isn't placed as it was before.